Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml

```
@@ -95,6 +95,7 @@
       - id: MedkitOxygenFilled
       - id: HolofanProjector
       - id: RCD
+      - id: RPD # Backmen-Edit-RPD
```

```suggestion
      - id: RPD # DS14-RPD
```
Если это всё не порт с Backmen